data/contrib_data.py

```diff
@@ -2,13 +2,17 @@
 
 import requests
 
-from community.git import get_org_name
-from data.models import Contributor
+from data.models import Contributor, Team
+from data.webservices import webservices_url
 
 
 def get_contrib_data():
     logger = logging.getLogger(__name__)
-    IMPORT_URL = 'https://webservices.' + get_org_name() + '.io/contrib/'
+    # IMPORT_URL = 'https://webservices.' + get_org_name() + '.io/contrib/'
+
+    # Using temporary rawgit link until the webservices is down
+    IMPORT_URL = ('https://rawgit.com/' + 'sks444/' + webservices_url() +
```

Comment:
`IMPORT_URL = community.git.webservices_url('contrib')`

`community.git.webservices_url(api_path)` must return a valid URL, either the real webservices endpoints or your fake one.

all of the mess to create the correct URL must go in `webservices_url`, hidden away from business logic